implementation/runtime/src/application_impl.cpp

```diff
@@ -1530,7 +1530,11 @@ std::shared_ptr<configuration> application_impl::get_configuration() const {
 }
 
 std::shared_ptr<policy_manager> application_impl::get_policy_manager() const {
+#ifndef VSOMEIP_DISABLE_SECURITY
     return configuration_->get_policy_manager();
+#endif
+    VSOMEIP_ERROR << __func__ << ": Must not be called with security disabled.";
```

Comment:
Shouldn't this be a `VSOMEIP_FATAL`?

Reply:
i will check this

Reply:
@kheaactua I ended up changing this to a warning instead of an error, since trying to get the manager isn't exactly an error, but could lead to one if this case is not properly handled. 
If in some instance you use it, it shouldn't create any big issues